lib/modules/platform/bitbucket/readme.md

```diff
@@ -23,6 +23,9 @@ Let Renovate use your app password by doing _one_ of the following:
 Remember to:
 
 - Set the `username` for the bot account
+<!-- prettier-ignore -->
+!!! warning "Use the actual username not your emailaddress"
+    Bitbucker cloud requires your actual username and not your emailaddress. You can find your actual username by going to your settings using the cog. It should be listed as under "Username" in the section "Bitbucket profile settings"
```

Comment:
```suggestion
- Set the `username` for the bot account (which is not your email address). You can find your username under the "Bitbucket profile settings" page for your account.
```

Reply:
Suggestion for the suggestion... :upside_down_face: 

We should probably _emphasize_ the _not_ part in: `(which is not your email address)`, to hammer the point home.

Reply:
I agree with suggested changes but would like to extend it a little to make the location more easily found since it is not at the first level, you need to click twice: first clik on the cog, then on "Personal Bitbucket Settings" which will land you on the  "Account settings" where you will be able to find the username.

Reply:
Updated the PR with my suggestion.